openlibrary/templates/lib/nav_foot.html

```diff
@@ -47,6 +47,17 @@ <h2>$:_('Help')</h2>
           <a class="footer-icons__github" href="https://github.com/internetarchive/openlibrary">github</a>
         </aside>
       </div>
+      <div id="footer-locale-menu">
+        <h2>$:_('Change Website Language')</h2> 
+        <ul id="locale-options">
+          <li><a href="#" data-lang-id="cs" title="Čeština">Čeština (cs)</a> </li>
+          <li><a href="#" data-lang-id="de" title="Deutsch">Deutsch (de)</a> </li>
+          <li><a href="#" data-lang-id="en" title="English">English (en)</a></li>
+          <li><a href="#" data-lang-id="es" title="Español">Español (es)</a></li>
+          <li><a href="#" data-lang-id="fr" title="Français">Français (fr)</a></li>
+          <li><a href="#" data-lang-id="te" title="తెలుగు">తెలుగు (te)</a></li>
```

Comment:
- Thinking about it some more, I think I like your original approach best of keeping the translated language name in the title. That's also what Wikipedia does.
- Added Croatian
```suggestion
          <li><a href="#" data-lang-id="cs" title="$_('Czech')">Čeština (cs)</a> </li>
          <li><a href="#" data-lang-id="de" title="$_('German')">Deutsch (de)</a> </li>
          <li><a href="#" data-lang-id="en" title="$_('English')">English (en)</a></li>
          <li><a href="#" data-lang-id="es" title="$_('Spanish')">Español (es)</a></li>
          <li><a href="#" data-lang-id="fr" title="$_('French')">Français (fr)</a></li>
          <li><a href="#" data-lang-id="hr" title="$_('Croatian')">Hrvatski (hr)</a></li>
          <li><a href="#" data-lang-id="te" title="$_('Telugu')">తెలుగు (te)</a></li>
```